app/models/zone.rb

```diff
@@ -78,6 +107,11 @@ def self.default_zone
     in_my_region.find_by(:name => "default")
   end
 
+  # Zone for paused providers (no servers in it), not visible by default
+  cache_with_timeout(:maintenance_zone) do
+    MiqRegion.my_region&.maintenance_zone
```

Comment:
Should you be using `miq_region.maintenance_zone` for consistency [here](https://github.com/ManageIQ/manageiq/pull/17452/files#diff-1e1418a05e4c9116c15a7b073e0b00bbR246) ?  looks strange to have two different ways to access the zone's region, specifically one with `&` and one without

Reply:
@agrare I don't like it in other place, don't like it to remove here. It'll fail lots of specs due to validation in EMS, which calls `Zone.maintenance_zone`. 
I don't want to fix lots of failing specs due to this cosmetic issue (see this PR's validation in EMS) and waste lots of useless time for this!

Reply:
duplicate of https://github.com/ManageIQ/manageiq/pull/17452#discussion_r228924571
followign here